### PR TITLE
qemu.tests.mq_change_qnum: Improve the method for getting ext_host

### PR DESCRIPTION
**Pull request by spcui**
Use get_host_default_gateway() to get ext_host

Signed-off-by: Shuping Cui <scui@redhat.com>

ID: 1242739

### CODE REVIEW

**Comment by spcui** [comment]
related on:
https://github.com/autotest/virt-test/pull/2272

**Comment by xutian** [comment]
Acked-by: Xu Tian

**Comment by CongSmile** [comment]
Acked-by: Cong Li <coli@redhat.com>

File: qemu/tests/mq_change_qnum.py [hunk 1]
```diff
@@ -179,13 +179,11 @@ def ping_test(dest_ip, ping_time, lost_raito, session=None):
...
             default_host = "www.redhat.com"
-            ext_host_get_cmd = params.get("ext_host_get_cmd", "")
             try:
-                ext_host = utils.system_output(ext_host_get_cmd)
-            except error.CmdError:
-                logging.warn("Can't get specified host with cmd '%s',"
-                             " Fallback to default host '%s'",
-                             ext_host_get_cmd, default_host)
+                ext_host = utils_net.get_host_default_gateway()
+            except Exception:
```
Comment (xutian) [inline qemu/tests/mq_change_qnum.py]: Why change to exception ?

**Pull request merged**